### PR TITLE
Add support for PHP8

### PR DESCRIPTION
**Pull request by WouterSioen**
This will allow this library to be installed on PHP8 as well

### CODE REVIEW

**Review by dbu — APPROVED** [review]
thanks!

makes me realize that we should have a ci step that lints the interfaces with the supported PHP versions. i checked the interfaces on 3v4l.org manually now, and as expected there seem to be no BC breaks in PHP 8 that affect us here.

**Comment by WouterSioen** [comment]
Yeah, I also verified it manually here. If you want, I can add a github action to this pr that runs php -l on all files in all supported php versions?

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.0|^8.0",
```
Comment (Nyholm) [inline composer.json]:
```suggestion
        "php": "^7.0 || ^8.0",
```

**Review by Nyholm — APPROVED** [review]
Thank you. I agree with this. 

Could you use the official syntax though?

**Comment by dbu** [comment]
thanks! tagged a 1.0.1

**Pull request merged**